src/dialog.html

```diff
@@ -1,6 +1,9 @@
 <html>
 <head>
-		
+	
+	<!-- <link rel="stylesheet" href="//build.origami.ft.com/bundles/css?modules=o-forms@^2.0.0,o-buttons@^3.0.3" /> -->
```

Comment:
I think it's useful to have the URL we used to generate the shrinkwrapped URL available.